### ISSUE TITLE
New Laptop ?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/hq#177]
# Why?

> as a **person joining** the @DWYL team on a ***full time*** basis ⏳ 
I **need a laptop** to get my work done 💻 
**so that** I can **succeed** in my quest to find the **meaning of life** and fulfil my **destiny**. ❤️  ✅  

# What?

Some people don't _have_ a laptop.
Others _prefer_ to have a different device for work and personal use.
What ever the reason, we need to make _a_ device ***available*** to people joining DWYL.

## Do Developers _Need_ Mac Laptops?

Through discussion we determined that our "Standard" Laptop does _not_ ***need*** to be a Mac.
( _just because a few [Developers use Macs](https://www.quora.com/Why-do-most-professional-programmers-prefer-Macs) and we have an observation bias towards them, does not mean that **all devs use macs** by any means. in fact by definition/stats _most_ devs don't use them!_ )

99% of the time developers are only using:
+ Web Browser(s)
+ Terminal
+ Text Editor
+ Local Web Server
+ Google Hangouts (_for remote team meetings_)
+ Occasional Image Editing for documenting or illustrating something.

None of these require a "_high powered_" Laptop.
In fact it could be argued that it's _wasteful_ to have a Mac Laptop if _all_ you are doing is writing code... 🤔 

## Chromebook ?

In case you have _not_ heard of them, watch/read: https://www.google.com/chromebook/

A Acer with 14" Full HD (1080p) Screen and **12 hour Battery Life** and 4GB RAM is ***£280***
http://www.currys.co.uk/gbuk/computing/laptops/laptops/acer-14-cb3-431-full-hd-chromebook-silver-10148244-pdt.html
![currys-acer-14-hd-chromebook](https://cloud.githubusercontent.com/assets/194400/22341607/5a392ba4-e3e9-11e6-995e-81f08074631e.png)

I actually _prefer_ to buy from a _physical_ retailer for "_peace of mind_" but for _comparison_,
the Amazon price is **£349.99**: https://www.amazon.co.uk/d/Computers-Accessories/Acer-CB3-431-Chromebook-Celeron-Storage/B01J3UVWKO/
![acer-chromebook-14-full-hd](https://cloud.githubusercontent.com/assets/194400/22341593/505087ea-e3e9-11e6-917d-6308fcff63db.png)

Top 10: Reasons To Buy A Chromebook: https://youtu.be/ebqDVB2xYXo
The Verge: Best Chromebook (2016) > https://youtu.be/Azzmbm17u-A 
(_comparison video. inconclusive. recommends [Acer Chromebook R11](http://www.argos.co.uk/product/6738589) which has a _low-res screen_
or [Dell "13 which is discontinued!](http://www.dell.com/uk/business/p/chromebook-13-7310/pd)_)

## "Cheap" MacBook Air

https://www.groupon.co.uk/deals/apple-refurbished-macbook-md508
![image](https://cloud.githubusercontent.com/assets/194400/22622126/546bcbbc-eb2a-11e6-8c09-a9ab0bd6cf17.png)

# tl;dr

Did you know: not _everyone_ has a Laptop? 😮 

We _often_ ***assume*** that _everyone_ already has a Laptop or at least a device they can use to start learning/working with. This is _obviously_ not the case for ***most people*** in the world Today (January 2017)! 

## Fun Fact (_Numbers_) of the day

There are 3.55 Billion People with Internet Access: http://www.internetlivestats.com/internet-users/
![number-of-internet-users](https://cloud.githubusercontent.com/assets/194400/22338946/7a4ff430-e3e0-11e6-8a0c-a6984cc64e67.png)

There are 7.48 Billion people in the world:
![number-of-people-in-the-world](https://cloud.githubusercontent.com/assets/194400/22339118/dc89da44-e3e0-11e6-9445-bdf85eaed888.png)

So _fewer_ than _half_ the people in the world have Internet Access.
And roughly [_half_ the people with Internet access are on Facebook ...](https://zephoria.com/top-15-valuable-facebook-statistics) (_so there's still **hope** for humanity!_) 😉 

http://www.worldometers.info/computers/
![image](https://cloud.githubusercontent.com/assets/194400/22339969/b4204978-e3e3-11e6-98ef-a4232fb8a215.png)

So... an _interesting_ question would be: 
given the rate of production for computers vs. the rate of _human_ reproduction,
and given that many computers are purchased to replace "old" or "broken" ones
will the number of computers catch up to the number of people in the world?

_obviously_ there are *way* more mobile/smart phones than "computers":
https://www.statista.com/statistics/330695/number-of-smartphone-users-worldwide/
![image](https://cloud.githubusercontent.com/assets/194400/22344477/47195c50-e3f4-11e6-86c2-6fd64e115895.png)

**Comment by sohilpandya** [comment]
@nelsonic we've had a lot of people go through F&C on chromebooks! They are awesome little devices. 😍 

Although there is a bit of tinkering required to get linux up and running on one of these. Have you ever tried https://c9.io can we recommend that people keep with chromeOS and use a cloud based IDE?

**Comment by sofer** [comment]
Acer Chromebook 14. Close to perfection, apparently

http://www.voltron00x.com/?p=1972

**Comment by nelsonic** [comment]
@sohilpandya yeah, I used C9 a lot when I got my first ChromeBook back in 2012 ... 
but went back to a Mac because the "_linux-on-chromebook_" story wasn't great at the time ...
But I get the feeling Chromebooks are _way_ better now in 2016 ...
Considering you can run Android Apps on the Chromebook I _suspect_ they are going to continue to pull ahead of the Windows Alternatives ... 
Also, the price difference is _nuts_: http://www.laptopsdirect.co.uk/st/refurbished-laptops
![image](https://cloud.githubusercontent.com/assets/194400/22412378/c2192ddc-e6a5-11e6-8b3a-cb0c5a4b83a3.png)

**Comment by sofer** [comment]
![acer](https://cloud.githubusercontent.com/assets/37059/22423212/654245d0-e6a5-11e6-8654-2eabb7121f35.jpg)

**Comment by sofer** [comment]
Note: there's no SD slot in the new Acer chromebooks. Also, 2GB is not a lot of memory. I am pretty happy with the one @nelsonic mentions at the top of this issue (despite having to temporarily turn off syncing, because of a rogue extension causing chrome to crash). I just picked up from PC World.

**Comment by nelsonic** [comment]
Orderededed one:
![image](https://cloud.githubusercontent.com/assets/194400/23827026/d0ba8888-06a1-11e7-8df8-a39fe83b6143.png)
Will do an "unboxing" and "dev setup" video next weekend. 😉

**Comment by Jbarget** [comment]
Just putting this here cos I was really surprised by how cheap a good laptop can be. A few of us from Nazareth got together to buy this for a student who doesn't have their own laptop. 2nd hand, but if you're looking for a few machines to have as 'spares':


![screen shot 2017-03-28 at 10 55 25](https://cloud.githubusercontent.com/assets/11725595/24399605/35b16f3a-13a5-11e7-9360-ceb790e7e44e.png)

![screen shot 2017-03-28 at 10 55 16](https://cloud.githubusercontent.com/assets/11725595/24399611/383bbf76-13a5-11e7-89a2-611a69c4a212.png)

**Comment by nelsonic** [comment]
@Jbarget a "refurbished" Lenovo is the way to go for people who are frugal/budget-concious. ✅

**Comment by iteles** [comment]
This arrived and we can now close this issue (seems to have been forgotten about 😊 ).
